ql/src/test/org/apache/hadoop/hive/ql/security/authorization/plugin/metastore/TestHiveMetaStoreAuthorizer.java

```diff
@@ -283,4 +284,50 @@ public void testM_DropCatalog_SuperUser() throws Exception {
       // no Exceptions for superuser as hive is allowed CREATE CATALOG operation
     }
   }
+
+  @Test
+  public void testN__ShowDatabase_authorizedUser() throws Exception {
```

Comment:
Please use CamelCase method names. We inherited a few places with different formatting, but new methods should stick to that

Reply:
did above 2